app/exporters/formatters/manual_indexable_formatter.rb

```diff
@@ -5,6 +5,10 @@ def type
     "manual"
   end
 
+  def indexable_attributes
+    super.merge({specialist_sectors: specialist_sectors})
```

Comment:
ManualIndexableFormatter's superclass already merges an overwritable empty-by-default hash called `extra_attributes`... I _think_ that's where this extra stuff should be defined.

It might also be good to document that superclass feature in a quick separate commit. I'd say do that by either

- Adding a comment to the `extra_attributes` method in the superclass, or
- Replacing the superclass' implementation with a `raise NotImplementedException` and defining it as a method that returns an empty hash on all subclasses that don't currently have their own